### ISSUE TITLE
How can I cancel the cross validation process?

### ISSUE THREAD

**Issue by clk-ad** [KonradReuter/PVS#1]

**Comment by KonradReuter** [comment]
Hello, thank you for your question. From what I can remember, simply pressing Ctrl+C multiple times in a row should do the trick. Let me know if this already fixes your issue. Otherwise I will have a look at it again.

**Comment by clk-ad** [comment]
I want to train your method on my own dataset. I only have the training and testing sets, and there is no validation set. Also, the validation set section is not included in the dataset structure directory you provided.

**Comment by KonradReuter** [comment]
Okay, now I understand. You will have to change the `train_loop` function in `train.py` then such that it works without a validation step. It should look something like this:

```python
def train_loop(
    train_dataloader: DataLoader, model: any, loss_fn: any, optimizer: any, args: dict, scheduler: any = None) -> None:
    """Performs training steps for the given number of epochs

    Args:
        train_dataloader (DataLoader): Dataloader for the train set
        model (any): Model to be trained
        loss_fn (any): Loss function
        optimizer (any): Optimizer
        args (dict): Arg dictionary
        scheduler (any, optional): Learning rate scheduler. Defaults to None.
    """
    logger.info(f"Training with {len(train_dataloader.dataset)} images.")
    # for each epoch perform a train and a validation step
    for e in range(args["epochs"]):
        train_loss = train_step_masked(train_dataloader, model, loss_fn, optimizer, args)
        logger.info(
            f"Train epoch {e}: train_loss: {train_loss}"
        )
        if args["use_wandb"]:
            wandb.log({"train_loss": train_loss, "train_epoch": e})
        if scheduler:
            scheduler.step()
```

You also have to change the function call in `main.py` in line 140, so it looks like this:
```python
train_loop(dataloaders["masked_vid_train"], model, loss, optimizer, args, scheduler=scheduler)
```

>Also, the validation set section is not included in the dataset structure directory you provided.

The validation set is selected as a subset of the training set during cross validation. Please have a look at line 470-483 in `scripts/dataloader.py` to see how this is done. I hope that makes it clear. Otherwise please clarify what your doubt is.

**Comment by KonradReuter** [comment]
I realized that there was a reference to a non-existing dataset called "SUN-SEG/ValidationDataset". Maybe this is what led to confusion, I removed it now.
